bears/css/CSSLintBear.py

```diff
@@ -17,7 +17,7 @@ class CSSLintBear:
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
-    CAN_DETECT = {'Syntax', 'Simplification'}
+    CAN_DETECT = {'Syntax', 'Redundancy'}
```

Comment:
i quote from the website: ``problematic patterns or signs of inefficiency`` maybe it's optimization? i dont know.